src/Connection.cpp

```diff
@@ -21,8 +21,9 @@ Connection::Connection(const std::string &uri)
   // "radio://0/80/2M/E7E7E7E7E7" -> connect over radio
   // "radio://*/80/2M/E7E7E7E7E7" -> auto-pick radio
   // "radio://*/80/2M/*" -> broadcast/P2P sniffing on channel 80
+  // "radiobroadcast://0/80/2M -> broadcast to all crazyflies on channel 80
 
-  const std::regex uri_regex("(usb:\\/\\/(\\d+)|radio:\\/\\/(\\d+|\\*)\\/(\\d+)\\/(250K|1M|2M)\\/([a-fA-F0-9]+|\\*)(\\?[\\w=&]+)?)");
+  const std::regex uri_regex("(usb:\\/\\/(\\d+)|radio(broadcast)*:\\/\\/(\\d+|\\*)\\/(\\d+)\\/(250K|1M|2M)(\\/([a-fA-F0-9]+|\\*)(\\?[\\w=&]+)?)*)");
```

Comment:
This regex now allows to have address and URI flags for the radiobroadcast backend as well, e.g., `radiobroadcast://0/80/2M/E7E7E7E7E7?ackfilter=0` would still be accepted. I think it would be cleaner to add a new option, i.e., `"(usb:\\/\\/(\\d+)|radio:\\/\\/(\\d+|\\*)\\/(\\d+)\\/(250K|1M|2M)\\/([a-fA-F0-9]+|\\*)(\\?[\\w=&]+)?)|radiobroadcast:\\/\\/(\\d+|\\*)\\/(\\d+)\\/(250K|1M|2M)"`. Not pretty, but better error handling & reporting for malformed URIs.

Reply:
True, fair point 👍